ui/appui-react/src/appui-react/ui-items-provider/ProviderItem.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module UiProvider
+ */
+
+import { ProvidedItem } from "@itwin/appui-abstract";
+
+/** Properties for an item provided by UiItemsProvider.
+ * @beta
+ */
+export type ProviderItem = ProvidedItem; // eslint-disable-line deprecation/deprecation
```

Comment:
Validating that the name change is intentional (provide**r** vs provide**d**)

Reply:
Yes, I am proposing this rename.